src/DssSpell.t.sol

```diff
@@ -1246,6 +1106,116 @@ contract DssSpellTest is DSTest, DSMath {
 
         return price;
     }
+    
+    function giveTokens(DSTokenAbstract token, uint256 amount) internal {
+        // Edge case - balance is already set for some reason
+        if (token.balanceOf(address(this)) == amount) return;
+
+        for (int i = 0; i < 40; i++) {
```

Comment:
This is clever. We might need to increase this number, iirc at least one token we've seen sets aside the first 100 slots for storage in it's proxy, but I suppose we can edit it if we get a fail at some point.